pages/api/startHostedActivitySessionViaHostedPagesLink/[hostedPagesLinkId].ts

```diff
@@ -19,7 +19,8 @@ export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse<Data>
 ) {
-  const { hostedPagesLinkId } = req.query as StartHostedActivitySessionParams
+  const { hostedPagesLinkId, track_id, activity_id } =
```

Comment:
[Q] are we referencing the published ID of the track or the definition ID of the track?